README.md

```diff
@@ -1,151 +1,149 @@
+- new substantially extended package pick for Coq 8.13.2 (the original pick from 2021.02 is also available)
+- new beta pick for Coq 8.14+rc1 - as close as possible to the updated pick for Coq 8.13.2
+
+## Changes in 2021.02.2
 
 - support for opam 2.1.0 (which integrates the opam system dependency manager *depext* - this needed a few adjustments)
 - fix issues with Cygwin binutils
-- various minor fixes for the Snap package (support gappa, clightgen, ...)
+- various minor fixes for the snap package (support gappa, clightgen, ...)
 - various minor fixes to the Windows installer (add icon for CoqIDE, ...)
 - minor cleanup and improvements of the Coq Platform scripts
 - the versions of provided Coq packages are identical to 2021.09.0
 
-## Changes from 2021.09.0 to 2021.09.0
+## Changes in 2021.02.1
 
 - added DMG package / installer for macOS
 - Coq and CoqIDE update to version 8.13.2 (bugfix release)
 - VST updated to version 2.7.1 (bugfix release)
 - new package `coq-hott` *The Homotopy Type Theory library*
 
-# Usage of the 2021.09.0 release
-
-Please refer to the ReadMe file for your OS.
-
-- macOS: see [README_macOS](README_macOS.md).
-- Windows: see [README_Windows](README_Windows.md)
-- Linux: see [README_Linux](README_Linux.md).
+</details>
 
-## Installation of additional packages or package variants
+<details><summary><font size="+1">Features of the Coq Platform</font></summary>
 
-### CompCert and VST variants
-
-For some packages, notably CompCert and VST (the Princeton tool-chain for verification of C code), exist various variants.
-By default the 64 bit variant of CompCert and the 64 bit variant of VST are installed.
+- fully opam based, also on Windows
+- single script call to install system dependencies, opam (if not there), a fresh opam switch and the Coq Platform
+- interactive (well script based) guidance of the user through the few parameters
+- one unified setup script for Windows, macOS and Linux with few OS dependent sections only
+- for Windows there is an additional wrapper batch script to setup Cygwin as build and working environment
+- for Windows there is in addition a classic Windows installer mostly intended for quick installation by beginners
+- for macOS a signed (but currently not yet notarized) DMG package is provided, also mostly intended for beginners
+- for Linux snap packages are provided via the Snap Store
+- it is easy to build variants of the provided installers with modified content
+- it is supported to install several versions of Coq in parallel - each will create a separate opam switch - this is intended e.g. for porting Coq developments from older versions of Coq
+- system prerequisites are installed using opam depext in a system independent manner
+- the script should be fairly robust and safe - it will immediately abort an all errors not explicitly handled
+- the script can be restarted if it fails - e.g cause of internet or memory issues - it will not redo things it already did
```

Comment:
Less colloquial:

```suggestion
- the script can be restarted if it fails - e.g because of internet or memory issues - it will not redo things it already did
```